docs/architecture.md

```diff
@@ -0,0 +1,223 @@
+# Architecture
+
+The core of OSP-Magnum is best described as a library that provides a set of functions and data structures intended to support a space flight simulator. The user is responsible for implementing their own application on the top level, allowing for full control over every feature.
```

Comment:
This might be scary to readers.

Maybe also point out that we have the test application providing an out of the box game, as well as the game engine itself.

Reply:
"The core of OSP-Magnum is best described as a library that provides a set of functions and data structures intended to support a space flight simulator. The user's top-level application has full control over every feature and their main loop. We provide a *Test Application* to act as an out-of-the-box game."